### ISSUE TITLE
bendctl can't load data into databend

### ISSUE THREAD

**Issue by wubx** [datafuselabs/databend#3111]
**Summary**
bendctl load data error 

$bendctl --databend_dir .databend --group local load $HOME/dataset/ontime.csv --table ontime
thread 'main' panicked at 'cannot post to http handler: reqwest::Error { kind: Request, url: Url { scheme: "http", cannot_be_a_base: false, username: "", password: None, host: Some(Ipv4(127.0.0.1)), port: Some(8000), path: "/v1/query", query: Some("wait_time=-1"), fragment: None }, source: hyper::Error(IncompleteMessage) }', cli/src/cmds/queries/query.rs:250:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**How to reproduce**
bendctl create cluster 

$bendctl --databend_dir .databend --group local query  $HOME/dataset/create_table.sql
[ok] ✅ Query precheck passed!

[ok] ✅ read rows: 0, read bytes: 0 B, rows/sec: 0 (rows/sec), bytes/sec: 0 (B/sec), time: 0.09 sec

$bendctl --databend_dir .databend --group local load $HOME/dataset/ontime.csv --table ontime
thread 'main' panicked at 'cannot post to http handler: reqwest::Error { kind: Request, url: Url { scheme: "http", cannot_be_a_base: false, username: "", password: None, host: Some(Ipv4(127.0.0.1)), port: Some(8000), path: "/v1/query", query: Some("wait_time=-1"), fragment: None }, source: hyper::Error(IncompleteMessage) }', cli/src/cmds/queries/query.rs:250:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

[How to get the server logs](https://databend.rs/development/how-to-get-server-logs/)

$curl --location --request POST '127.0.0.1:8000/v1/statement/' --header 'Content-Type: text/plain' --data-raw 'select version()'
{"id":"2675049b-33c9-4415-8d05-025552ffa3dd","columns":{"fields":[{"name":"version()","data_type":"String","nullable":false}],"metadata":{}},"data":[["DatabendQuery v-0.1.0-a69c4cf-simd(1.57.0-nightly-2021-11-25T17:01:36.054958+00:00)"]],"state":"Succeeded","error":null,"stats":{"progress":{"read_rows":1,"read_bytes":1,"total_rows_to_read":0}},"stats_uri":"/v1/query/2675049b-33c9-4415-8d05-025552ffa3dd","final_uri":"/v1/query/2675049b-33c9-4415-8d05-025552ffa3dd/kill?delete=true","next_uri":null}

$ps axu |grep databend
wubx             91157   0.0  0.2 34669720 154640 s006  S    12:04下午   0:17.68 /Users/wubx/.databend/local/bin/v0.5.27-nightly/databend-query
wubx             91156   0.0  0.0 34321408  17544 s006  U    12:04下午   0:02.83 /Users/wubx/.databend/local/bin/v0.5.27-nightly/databend-meta

**Comment by youngsofun** [comment]
```
2021-11-26T07:00:50.794794Z ERROR analyze{ctx.id="9b61d453-7594-41f4-9c24-40dcf8f2d81d"}: common_tracing::panic_hook: panicked at 'index out of bounds: the len is 109 but the index is 109', common/datavalues/src/data_schema.rs:67:10 panic.file="common/datavalues/src/data_schema.rs" panic.line=67 panic.column=10```

**Comment by youngsofun** [comment]
may bad row constructed in "values"

save error for:

```
mysql> insert into t1(a, b) values (1, 1, 1);
```

**Comment by wubx** [comment]
make stateful-test  get the same error log

**Comment by youngsofun** [comment]
1. ontime.csv is ill formatted: each line (including the header line) is ended with ",",   csv-reader return one more columns per row.
   2. but it does not cause err yesterday..
2. insert become very slow compared with  yesterday
3. during insert,   select count(*)  report error.

```
mysql> select count(*) from ontime;
ERROR 1105 (HY000): Code: 6, displayText = Unable to get field named "Quarter". Valid fields: ["Year"]
```

**Comment by zhang2014** [comment]
> during insert, select count(*) report error.

fixes by #3127
